src/cordova-util.js

```diff
@@ -1,5 +1,11 @@
 export const isCordova = () => !!window.cordova;
 
+export const isAndroid = () =>
+  isCordova() && window.cordova.platformId === 'android' ? true : false;
```

Comment:
This is a very small detail, but you don't need to use a ternary (conditional) operator here. The condition is already a boolean so you can leave out `? true : false`.